.gitignore

```diff
@@ -21,3 +21,4 @@ bundler_stubs
 bin
 gem-private_key.pem
 save_path_tmp
+vendor/bundle
```

Comment:
Why was this necessary?

Reply:
Sorry, I miss the reason. 🙏 

I prefer a style to use the [`bundle install --deployment`](https://bundler.io/v2.0/man/bundle-install.1.html#DEPLOYMENT-MODE) command, which installs the gems to `vendor/bundle` directory.
If you don't like it, I'll remove it. 👍

Reply:
Ok -- it's fine